### PR TITLE
Add backend-config option

### PR DESCRIPTION
**Pull request by drsherluck**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -101,7 +102,8 @@ async function terraform(terraformDirectory, args) {
   }
 
   core.startGroup('Run terraform init');
-  const tfi = await terraform(terraformDirectory, ['init']);
+  let tfi_args = !terraformBackend ? ['init'] : ['init', '-backend-config=' + terraformBackend, '-reconfigure'];
```
Comment (tun0) [inline index.js:105]: I doubt `-reconfigure` is needed here? It runs in a clean environment.

**Comment by drsherluck** [comment]
https://github.com/Pararius/frontend/runs/7671697596?check_suite_focus=true <--- test run

File: index.js (hunk 1 above)
Comment (drsherluck) [reply index.js]: removed it 👍🏻

File: index.js [hunk 2]
```diff
@@ -101,7 +102,8 @@ async function terraform(terraformDirectory, args) {
   }
 
   core.startGroup('Run terraform init');
-  const tfi = await terraform(terraformDirectory, ['init']);
+  let tfi_args = !terraformBackend ? ['init'] : ['init', '-backend-config=' + terraformBackend];
```
Comment (tun0) [inline index.js:105]: Feels a bit clunky to have `init` there twice. But, all alternatives I can think of aren't that much better :thinking:

File: index.js [hunk 3]
```diff
@@ -101,7 +102,8 @@ async function terraform(terraformDirectory, args) {
...
+  const tfi = await terraform(terraformDirectory, tfi_args);
```
Comment (tun0) [inline index.js:106]: Perhaps something like this?

```suggestion
  let tfi_args = !terraformBackend ? [] : ['-backend-config=' + terraformBackend];
  const tfi = await terraform(terraformDirectory, ['init'].concat(tfi_args));
```

**Review by tun0 — APPROVED** [review]

**Pull request merged**